### ISSUE TITLE
FR: Option to only show dangling links in the current note

### ISSUE THREAD

**Issue by chrisgrieser** [graydon/obsidian-dangling-links#14]
pretty much what the title says :)

Thanks for the useful plugin, really helpful after migrating from a different note taking app to find the links that do not work anymore.

**Comment by etiennepellegrini** [comment]
I agree that this option would be extremely valuable, allowing one to fix dangling links as they arise when editing files.

**Comment by claremacrae** [comment]
Hi @chrisgrieser and @etiennepellegrini,

Just checking my understanding, as the description can read at least two ways.

I read this as "make the current Dangling Links panel restrict its results to only any links that are in the currently focussed note".

**Comment by claremacrae** [comment]
> Hi @chrisgrieser and @etiennepellegrini,
> 
> Just checking my understanding, as the description can read at least two ways.
> 
> I read this as "make the current Dangling Links panel restrict its results to only any links that are in the currently focussed note".

Is that what you meant when you wrote or up-voted this?

**Comment by chrisgrieser** [comment]
@claremacrae yes, exactly. (Though I must say that I don't use the plugin anymore by now, so no need from my side to implement this anymore…)

**Comment by etiennepellegrini** [comment]
That's also what I meant :-) I would still like to have this capability, if possible!

**Comment by claremacrae** [comment]
Thanks for the replies, folks...

I've edited the summary to try and remove the ambiguity ...

(the alternative interpretation is that it was requesting a widget to be added at the bottom of each pane, which showed any dangling links in that note... similar to an option in Obsidian for showing backlinks within the pane... - and which would likely be rather more work...)
